client/src/api/repo.api.ts

```diff
@@ -1,5 +1,6 @@
 import { AxiosResponse } from "axios";
 import { axiosInstance } from "../util/http";
+import { dot } from "node:test/reporters";
```

Comment:
This isn't necessary. I'm guessing you mistyped `dto` and your IDE imported `dot`.